[Bug]: AttributeError: module 'cmd' has no attribute 'Cmd'

### Describe the bug

```
 Please wait a moment, download file [/tmp/ob_log_193.22.146.60_20240703110949_20240703114049/election.log] from server 193.22.146.60 to [/root/obdiag_analyze_pack_202
40703113949/193_22_146_60/election.log]
[2024-07-03 11:39:52.157] [ERROR] Running Error: module 'cmd' has no attribute 'Cmd'
[2024-07-03 11:39:52.157] [ERROR] Traceback (most recent call last):
[2024-07-03 11:39:52.158] [ERROR]   File "/root/obdiag/cmd.py", line 222, in do_command
[2024-07-03 11:39:52.158] [ERROR]     ret = self._do_command(obdiag)
[2024-07-03 11:39:52.158] [ERROR]           ^^^^^^^^^^^^^^^^^^^^^^^^
[2024-07-03 11:39:52.158] [ERROR]   File "/root/obdiag/cmd.py", line 603, in _do_command
[2024-07-03 11:39:52.158] [ERROR]     return obdiag.analyze_fuction('analyze_log', self.opts)
[2024-07-03 11:39:52.158] [ERROR]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-07-03 11:39:52.158] [ERROR]   File "/root/obdiag/core.py", line 261, in analyze_fuction
[2024-07-03 11:39:52.158] [ERROR]     handler.handle()
[2024-07-03 11:39:52.158] [ERROR]   File "/root/obdiag/handler/analyzer/analyze_log.py", line 142, in handle
[2024-07-03 11:39:52.158] [ERROR]     handle_from_node(node)
[2024-07-03 11:39:52.158] [ERROR]   File "/root/obdiag/handler/analyzer/analyze_log.py", line 137, in handle_from_node
[2024-07-03 11:39:52.158] [ERROR]     resp, node_results = self.__handle_from_node(node, local_store_parent_dir)
[2024-07-03 11:39:52.158] [ERROR]                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-07-03 11:39:52.158] [ERROR]   File "/root/obdiag/handler/analyzer/analyze_log.py", line 208, in __handle_from_node
[2024-07-03 11:39:52.159] [ERROR]     self.stdio.start_loading('analyze log start')
[2024-07-03 11:39:52.159] [ERROR]   File "/root/obdiag/stdio.py", line 577, in start_loading
[2024-07-03 11:39:52.159] [ERROR]     self.sync_obj = self._start_sync_obj(IOHalo, lambda x: x.stop_loading('fail'), *arg, **kwargs)
[2024-07-03 11:39:52.159] [ERROR]                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-07-03 11:39:52.159] [ERROR]   File "/root/obdiag/stdio.py", line 551, in _start_sync_obj
[2024-07-03 11:39:52.159] [ERROR]     raise e
[2024-07-03 11:39:52.159] [ERROR]   File "/root/obdiag/stdio.py", line 547, in _start_sync_obj
[2024-07-03 11:39:52.159] [ERROR]     self.sync_obj = sync_clz(*arg, **kwargs)
[2024-07-03 11:39:52.159] [ERROR]                     ^^^^^^^^^^^^^^^^^^^^^^^^
[2024-07-03 11:39:52.159] [ERROR]   File "/root/obdiag/stdio.py", line 244, in __init__
[2024-07-03 11:39:52.159] [ERROR]     super(IOHalo, self).__init__(text=text, color=color, text_color=text_color, spinner=spinner, animation=animation, placement=placement, interval=interval, enabled=en
abled, stream=stream)
[2024-07-03 11:39:52.159] [ERROR]   File "/opt/mise/installs/python/3.12.4/lib/python3.12/site-packages/halo/halo.py", line 100, in __init__
[2024-07-03 11:39:52.159] [ERROR]     environment = get_environment()
[2024-07-03 11:39:52.159] [ERROR]                   ^^^^^^^^^^^^^^^^^
[2024-07-03 11:39:52.159] [ERROR]   File "/opt/mise/installs/python/3.12.4/lib/python3.12/site-packages/halo/_utils.py", line 44, in get_environment
[2024-07-03 11:39:52.159] [ERROR]     from IPython import get_ipython
[2024-07-03 11:39:52.159] [ERROR]   File "/opt/mise/installs/python/3.12.4/lib/python3.12/site-packages/IPython/__init__.py", line 54, in <module>
[2024-07-03 11:39:52.160] [ERROR]     from .core.application import Application
[2024-07-03 11:39:52.160] [ERROR]   File "/opt/mise/installs/python/3.12.4/lib/python3.12/site-packages/IPython/core/application.py", line 26, in <module>
[2024-07-03 11:39:52.160] [ERROR]     from IPython.core import release, crashhandler
[2024-07-03 11:39:52.160] [ERROR]   File "/opt/mise/installs/python/3.12.4/lib/python3.12/site-packages/IPython/core/crashhandler.py", line 27, in <module>
[2024-07-03 11:39:52.160] [ERROR]     from IPython.core import ultratb
[2024-07-03 11:39:52.160] [ERROR]   File "/opt/mise/installs/python/3.12.4/lib/python3.12/site-packages/IPython/core/ultratb.py", line 111, in <module>
[2024-07-03 11:39:52.160] [ERROR]     from IPython.core import debugger
[2024-07-03 11:39:52.160] [ERROR]   File "/opt/mise/installs/python/3.12.4/lib/python3.12/site-packages/IPython/core/debugger.py", line 129, in <module>
[2024-07-03 11:39:52.160] [ERROR]     from pdb import Pdb as OldPdb
[2024-07-03 11:39:52.160] [ERROR]   File "/opt/mise/installs/python/3.12.4/lib/python3.12/pdb.py", line 215, in <module>
[2024-07-03 11:39:52.160] [ERROR]     class Pdb(bdb.Bdb, cmd.Cmd):
[2024-07-03 11:39:52.160] [ERROR]                        ^^^^^^^
[2024-07-03 11:39:52.160] [ERROR] AttributeError: module 'cmd' has no attribute 'Cmd'
```

### Environment

python 3.12.4

### Fast reproduce steps

obdiag analyze log

### Expected behavior

_No response_

### Actual behavior

_No response_

### Additional context

我感觉就是cmd.py 和全局的cmd模块名字冲突

--- a/dev_init.sh
+++ b/dev_init.sh
@@ -42,6 +42,8 @@ copy_file(){
     mkdir -p ${OBDIAG_HOME}/gather
     if [ -d "${WORK_DIR}/handler/checker/tasks" ]; then
         cp -rf ${WORK_DIR}/handler/checker/tasks  ${OBDIAG_HOME}/check/
+        cp -rf ${WORK_DIR}/handler/checker/tasks/obproxy_check_package.yaml  ${OBDIAG_HOME}/check/
+        cp -rf ${WORK_DIR}/handler/checker/tasks/observer_check_package.yaml  ${OBDIAG_HOME}/check/
     fi
     if [ -d "${WORK_DIR}/handler/gather/tasks" ]; then
         cp -rf ${WORK_DIR}/handler/gather/tasks  ${OBDIAG_HOME}/gather/
@@ -51,8 +53,8 @@ copy_file(){
         cp -rf ${WORK_DIR}/example  ${OBDIAG_HOME}/
     fi
 
-    if [ -d "${WORK_DIR}/rca" ]; then
-        cp -rf ${WORK_DIR}/rca  ${OBDIAG_HOME}/
+    if [ -d "${WORK_DIR}/handler/rca/scene" ]; then
+        cp -rf ${WORK_DIR}/handler/rca/scene  ${OBDIAG_HOME}/rca
     fi
 
 }

--- a/diag_cmd.py
+++ b/diag_cmd.py
@@ -11,7 +11,7 @@
 # See the Mulan PSL v2 for more details.
 
 """
-@file: cmd.py
+@file: diag_cmd.py
 @desc:
 """
 

--- a/main.py
+++ b/main.py
@@ -16,7 +16,7 @@
 """
 
 import sys
-from obcmd import MainCommand
+from diag_cmd import MainCommand
 from stdio import IO
 
 ROOT_IO = IO(1)

